worker-workflow/README.md

```diff
@@ -0,0 +1,36 @@
+# worker-workflow
+
+## Summary
+
+The Workflow Worker is used to create a script representing a workflow that can be executed against a document to send the document to another worker to have an action performed on it.
+
+## Implementation
+
+The Workflow Worker behaviour for a task message is as follows;
+
+- Retrieve the workflow referenced by document in task message by communicating with external processing API.
+- Transform the workflow to a JavaScript script that the document can be evaluated against.
+- Evaluate the document against the workflow script to mark the document with the next suitable action to perform on the document e.g. send to Language Detection Worker, and modify the response options on the document to direct it to the appropriate queue.
+- Store the transformed workflow script on the document as a script to be executed by Document Workers during post processing. The intention is that once the next worker has completed its action it will evaluate the document against the workflow again and determine the next action to execute, sending the document to that next worker until all actions on the workflow are completed.
+
+## Configuration
+
+The worker utilizes the following configuration properties. The environment variables to control these and defaults values can be seen in the container for the worker, [here](../worker-workflow-container).
+
+### processingApiUrl
```

Comment:
I'd like to avoid duplicating that environment variable information. The keys are mentioned here just to give some description, how about if I move the key details into that configuration table on the container page and link to the configuration from here?

Reply:
Added 'description' column to the container config properties table and moved text there.